### ISSUE TITLE
Multi Model X-RAY Structures (2Q4M)

### ISSUE THREAD

**Issue by lafita** [rcsb/symmetry#90]
If a structure is X-RAY and multi-model, and not a biological assembly, all the models are considered for symmetry, so identity symmetry axes are returned (and a lot of levels). A test case is **2Q4M**.

This bug was introduced when providing support for biological assemblies, so a check should be made that the structure is a BA before using all models.

**Comment by josemduarte** [comment]
> so a check should be made that the structure is a BA before using all models.

Or use the multi-chain bioassemblies

**Comment by lafita** [comment]
I agree Jose, but we have the problem that we cannot display them in **Jmol**. Definitely it will have to be changed when **Jmol** allows passing structures as _MMCIF_.

**Comment by josemduarte** [comment]
Oh jmol... forgot that :) Passing them as mmCIF to jmol should be quite easy now that we have mmCIF writers, shouldn't it?

**Comment by lafita** [comment]
Yes! Everything is ready from **biojava** side, but the latest **Maven** version of **Jmol** is a very old one, with buggy support for _mmCIF_. We either request for an update of their **Maven** version (which I think we already did), or we move away from **Jmol**.

**Comment by josemduarte** [comment]
Of course the maven problem... forgot that too! sorry about that.

Yes getting the latest jmol should be a priority... It even has a MMTF reader... perhaps we could even try to pass structures as MMTF.
